controllers/usersController.js

```diff
@@ -48,6 +48,25 @@ const getUser = async (req, res) => {
   }
 }
 
+/**
+ * Checks whether a user exists or not
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const checkUser = async (req, res) => {
+  try {
+    const result = await userQuery.fetchUser({ username: req.params.username })
+    return res.json({
+      userAvailable: !result.userExists
```

Comment:
Should be `isUsernameAvailable` as per API contracts

Reply:
done